### ISSUE TITLE
[IP #125] Implement Measurement Order

### ISSUE THREAD

**Issue by RomanLangrehr** [Beagle-PSE/Beagle#610]
Implement MeasurementOrder

Depends on: IP#040 (SEFF Classes). ( #571 )

**Comment by RomanLangrehr** [comment]
+MeasurementEvents +LaunchConfigutation.

**Comment by acBerger** [comment]
I start testing this.

**Comment by RomanLangrehr** [comment]
Okay. The measurementEventParser is not quite finished yet, but I'll do this soon.

**Comment by acBerger** [comment]
This should not be a problem.

**Comment by acBerger** [comment]
I (hopefully) added and pushed tests for all classes except `MeasurementEventParser`.
I am on implementing this one but as I confusing myself often it is likely that this will still take some time.

**Comment by RomanLangrehr** [comment]
Implementation is done now.

The LaunchConfigurations have to be reworked, but I think it can get more complicated than just adding JVM arguments and the classpath, so I would appreciate creating a new issue for that.

Please carfully test `MeasurementEventParser#getMeasurementResultsFor(SeffLoop)` because it is pretty complex do determine, which body-executions go together.

The test `ParameterisationDependentMeasurementResultTest#equals` must be adapted, because we decided that each Measurement Result is only equal to it's own instance.

**Comment by acBerger** [comment]
![externalcallp](https://cloud.githubusercontent.com/assets/10773356/12871888/057008ca-cd8f-11e5-86f1-384dd21f1a6f.png)
You should return an empty Set here. First this is the logical consequence of not having any results, second the Javadoc defines there is never `null` returned.

**Comment by RomanLangrehr** [comment]
> You should return an empty Set here. First this is the logical consequence of not having any results, second the Javadoc defines there is never `null` returned.

Done

**Comment by acBerger** [comment]
There should be tests for every class and method now. Some are failing and I am not sure if this is caused by a mistake I made or not. 
I still have to think of some more test cases for the `getMeasurementResultsFor` Methods. However, you can start checking your code with the help of the tests now.
Please let me know if I have to change something.

**Comment by acBerger** [comment]
For me it would be best if you can have a look at the tests, expecially the failing ones and tell me if I have to change something.

**Comment by jGleitz** [comment]
Why isn’t there a PR on this? What’s missing?

**Comment by RomanLangrehr** [comment]
Work in progress...

**Comment by RomanLangrehr** [comment]
@acBerger Test coverage is not 100% yet.
